src/pages/studyView/chartHeader/ChartHeader.tsx

```diff
@@ -361,6 +367,65 @@ export class ChartHeader extends React.Component<IChartHeaderProps, {}> {
                 </li>
             );
         }
+        if (
+            this.props.chartControls &&
+            this.props.chartControls.showViolinPlotToggle &&
+            this.props.toggleViolinPlot
+        ) {
+            items.push(
+                <li>
+                    <a
+                        className="dropdown-item violinCheckbox"
+                        onClick={this.props.toggleViolinPlot}
+                    >
+                        <FlexAlignedCheckbox
+                            checked={
+                                !!(
+                                    this.props.chartControls &&
+                                    this.props.chartControls.violinPlotChecked
+                                )
+                            }
+                            label={
+                                <span style={{ marginTop: -3 }}>
+                                    Show Violin Plot
+                                </span>
+                            }
+                            style={{ marginTop: 1, marginBottom: -3 }}
+                        />
+                    </a>
+                </li>
+            );
+        }
+
+        if (
+            this.props.chartControls &&
+            this.props.chartControls.showBoxPlotToggle &&
+            this.props.toggleBoxPlot
+        ) {
+            items.push(
+                <li>
+                    <a
+                        className="dropdown-item violinCheckbox"
+                        onClick={this.props.toggleBoxPlot}
+                    >
+                        <FlexAlignedCheckbox
+                            checked={
+                                !!(
+                                    this.props.chartControls &&
+                                    this.props.chartControls.boxPlotChecked
+                                )
+                            }
+                            label={
+                                <span style={{ marginTop: -3 }}>
```

Comment:
what do these negative margins do?

Reply:
it aligns the label to the checkbox properly